cmd/dev.go

```diff
@@ -13,6 +13,13 @@ import (
 	"github.com/spf13/cobra"
 )
 
+// devCmd is the command for playground-style development
+var devCmd = &cobra.Command{
+	Use:   man.Docs.GetDoc("dev").Use,
+	Short: man.Docs.GetDoc("dev").Short,
+	Long:  man.Docs.GetDoc("dev").Long,
+}
+
```

Comment:
You can use the GetCommand here so you don't have to map everything 

```suggestion
// devCmd is the command for playground-style development
var devCmd = man.Docs.GetCommand("dev")
```